### DISCUSSION TITLE
Trouble specifying board

### DISCUSSION THREAD

**Discussion by mamsterla** [bengtmartensson/Infrared4Arduino#91] [General]
I am using VSCode and PlatformIO with an ATMega2560 clone board.  I currently set the environment up as:

board = megaatmega2560

But I think that  there is a problem with I4A seeing this as an ATMega2560 board.  What is the correct way to force the board architecture for a simple program.  I am just testing RC5 sending and want to know the correct pin (which should be 9, but I am not getting through the sender initialization).

**Comment by bengtmartensson** [comment]
I personally am not currently using VSCode or PlatformIO, so I cannot help. Anyone? Another forum might be better for this question, since it has not so much with I4A to do.

**Comment by mamsterla** [comment]
Bengt:

I was able to figure it out.  I had another issue with the missing initialization of Serial (oh my) that caused a problem.  I was able to get it to work on the ATMega2560 clone and it is now flawless.  I appreciate the rapid turnaround and your work on this library.

-MA
